ratelimit_test.go

```diff
@@ -111,7 +134,7 @@ func (r *runnerImpl) goWait(fn func()) {
 }
 
 func Example() {
-	rl := ratelimit.New(100) // per second
+	rl := New(100) // per second
```

Comment:
optional: I like the idea of the example tests showing the full name (`ratelimit.New`) because these show as examples in the documentation. Consider moving them to an `example_test.go` file where we continue to use `package ratelimit_test`.

Reply:
Nice, me like.